### ISSUE TITLE
Error: Duplication of element resources.repository.web

### ISSUE THREAD

**Issue by szabgab** [ikluft/WebFetch#2]
I tried to setup GitHub Actions for this module but encountered this error. It is strange as it happens while trying to install the dependencies and it works and the tests pass on my own computer. The failure is only on the CI server. You can see the output here: https://github.com/szabgab/WebFetch/actions/runs/3296388412/jobs/5435987590
I 

I also encountered the same error earlier in other modules: https://github.com/sdondley/Mojo-DOM-Role-Analyzer/issues/1

**Comment by ikluft** [comment]
That is interesting. The error message "Duplication of element resources.repository.web at /opt/hostedtoolcache/perl/5.30.3/x64/lib/site_perl/5.30.3/Dist/Zilla.pm line 604" indicates it came from Dist::Zilla. Following from that line in its code, the error message comes from CPAN::Meta::Merge in the _uniq_map() function. It's getting "resources.repository.web" from CPAN::Meta::Spec, which should be the URL of the Github repository in this case. It isn't clear from this data what the values are that it's trying to merge and why they aren't the same. Investigation continues...

**Comment by ikluft** [comment]
Further digging looks like the Dist::Zilla plugins GithubMeta and MetaResources are creating conflicting metadata on that CI platform & Perl 5.30.3. I'm guessing it could be a string mismatch in the Github web URL generated by each plugin. Your link to the issue with the prior occurrence on Mojo-DOM-Role-Analyzer was helpful.

**Comment by ikluft** [comment]
I committed a fix - let me know how the CI system likes it. I found MetaResources was creating metadata redundant to GithubMeta. The difference was visible in the generated META.yml and META.json. I removed the MetaResources config section from dist.ini because its manual resources.repository.web setting couldn't coexist with GithubMeta's automatic resources.repository (no ".web"). GithubMeta needed a new "issues = 1" setting to tell it to generate a bugtracker link in the metadata.
